stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channels/header/ChannelListHeader.kt

```diff
@@ -57,6 +61,8 @@ public fun ChannelListHeader(
     title: String = "",
     currentUser: User? = null,
     connectionState: ConnectionState = ConnectionState.CONNECTED,
+    color: Color = ChatTheme.colors.barsBackground,
+    shape: Shape = RectangleShape,
```

Comment:
We shouldn't be using a material shape here, should be a ChatTheme shape. If we don't have one, feel free to add it!

Reply:
And we should probably update the channel list docs while we're at it